distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -1,4 +1,27 @@
-#Requires -Version 5.0
+#Requires -Version 4.0
+function DownloadFiles {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$DownloadSource,
+        [Parameter(Mandatory=$true)]
+        [string]$Outputfile
+    )
+    $client = New-Object System.Net.WebClient
+    try {
+        $client.DownloadFileAsync($DownloadLocation,$Outputfile)
+        while ($client.IsBusy)
+        {
+            Write-Host "Downloading file..."
+            Sleep -Seconds 3
+        }
+    }
+    catch [Net.WebException]
+    {
+        Write-Host $_
```

Comment:
I wouldn't simply catch the exception and let the script continue if there's an error with the download.